sim/aviron/src/lib/io.zig

```diff
@@ -134,7 +103,8 @@ pub const RAM = struct {
 pub const EEPROM = RAM; // actually the same interface *shrug*
 
 pub const IO = struct {
-    pub const Address = u6;
+    // Some AVR families (e.g., XMEGA) expose extended I/O up to 0xFFF (12 bits).
+    pub const Address = u12;
```

Comment:
`IO` should use `u6` here as it's the size of an IO address still.

The XMEGA 12 bit thingy isn't on the IO bus, but on the data bus (which has 24 bits)